miniapp/src/test/java/com/rakuten/tech/mobile/miniapp/display/RealMiniAppDisplayTest.kt

```diff
@@ -0,0 +1,70 @@
+package com.rakuten.tech.mobile.miniapp.display
+
+import android.content.Context
+import android.view.ViewGroup
+import android.webkit.WebView
+import androidx.test.core.app.ApplicationProvider.getApplicationContext
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.rakuten.tech.mobile.miniapp.TEST_BASE_PATH
+import kotlinx.coroutines.test.runBlockingTest
+import org.amshove.kluent.*
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import kotlin.test.assertTrue
+
+@RunWith(AndroidJUnit4::class)
+class RealMiniAppDisplayTest {
+
+    lateinit var context: Context
+
+    @Before
+    fun setup() {
+        context = getApplicationContext()
+    }
+
+    @Test
+    fun `for a given basePath, obtainView returns corresponding view to the caller`() =
+        runBlockingTest {
+            val display = Displayer(context).createMiniAppDisplay(basePath = TEST_BASE_PATH)
```

Comment:
This should be constructed manually instead of via `Displayer`. i.e. `RealMiniAppDisplay(context, basePath)`

Reply:
Yea, it got a bit mixed up after the second level refactor.